03-rnaseq/pathway-analysis_rnaseq_01_ora.html

```diff
@@ -4007,7 +4007,7 @@ <h2><span class="header-section-number">4.2</span> Import data</h2>
 <span id="cb31-3"><a href="#cb31-3"></a><span class="co"># desired gene list results</span></span>
 <span id="cb31-4"><a href="#cb31-4"></a>gene_module_df &lt;-<span class="st"> </span>readr<span class="op">::</span><span class="kw">read_tsv</span>(gene_module_url)</span></code></pre></div>
 <pre><code>## 
-## ── Column specification ──────────────────────────────────────────────────────────────────────────────
+## ── Column specification ────────────────────────────────────────────────────────────────────────────────────────────────────
```

Comment:
Commenting on this change because it reminded me of something that we should look at, which is whether we can make the text output width a bit narrower. When rendered, these lines often end up wrapped, which is not ideal. We could partially solve this by making sure output blocks do not wrap in the html, but a better solution would be to actually control the width, if possible.

Reply:
I think this is pertinent to #289